services/quizzes/src/components/exercise-service-views/ViewSubmission/index.tsx

```
@@ -243,212 +280,267 @@ const Submission: React.FC<React.PropsWithChildren<SubmissionProps>> = ({
+        const submitMessageOnly =
+          !lastFeedbackItemId &&
+          !componentDescriptor.shouldDisplayCorrectnessMessageAfterAnswer &&
+          item.id === lastSubmitMessageItemId &&
+          submitMessage ? (
+            <SubmissionFeedback
+              itemFeedback={itemAnswerFeedback}
+              itemModelSolution={itemModelSolution}
+              questionType={item.type as QuizItemType}
+              submitMessage={submitMessage}
+              showScore={false}
+            />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)